documentation/slate/source/includes/apis.md

```diff
@@ -733,7 +733,7 @@ coordinates, returning a [places](#place) collection.
   nop      | admin_uri[] | array of string | If filled, will filter the search within the given admin uris       |
   nop      | filter      | string          | Use to filter returned objects. for example: places_type.id=theater |
   nop      | disable_geojson | boolean     | remove geojson from the response  | False
-  nop      | add_poi_infos  | boolean      | Activate the output of additional infomations about the poi. For example, parking availability(BSS, car parking etc.) in the bicycle_rental, car parking pois of response     | [`bss_stands`, `car_park`]
+  nop      | add_poi_infos  | boolean      | Activate the output of additional infomations about the poi. For example, parking availability(BSS, car parking etc.) in the bicycle_rental, car parking pois of response. When an emtpy string is passed to this parameter, it deactivates all   | [`bss_stands`, `car_park`]
```

Comment:
"pass `add_poi_infos[]=&` (empty string) to deactivate all" would be clearer I think